installation/linux.md

```diff
@@ -61,60 +61,57 @@ First, add the openHAB 2 Bintray repository key to your package manager:
 wget -qO - 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab' | sudo apt-key add -
 ```
 
-As long as openHAB 2 is in its development state you may choose between the latest *Beta* release or a *Snapshot* build.
-The snapshot build is created [almost daily](https://bintray.com/openhab/apt-repo2/openhab2#release), always including the latest changes to the openHAB 2 core and add-ons.
-Decide between two options:
+Then, you can choose between, *Official (Stable)*, *Beta* or *Snapshot* builds:
 
-* **Beta Release**
+* **Official Release**
 
-  Add the **openHAB 2 Beta Repository** to your systems apt sources list (will overwrite the existing `openhab2.list`):
+  Contains a stable release with tested features passed the test of time from the builds below.
+  
+  Add the **openHAB 2 Stable Repository** to your systems apt sources list:
 
   ```shell
-  echo 'deb http://dl.bintray.com/openhab/apt-repo2 testing main' | sudo tee /etc/apt/sources.list.d/openhab2.list
+  echo 'deb http://dl.bintray.com/openhab/apt-repo2 stable main' | sudo tee /etc/apt/sources.list.d/openhab2.list
   ```
 
-  Resynchronize the package index:
+* **Beta Release**
 
-  ```shell
-  sudo apt-get update
-  ```
+  The beta builds come out less frequently, but will contain new features that are currently in the testing phase.
 
-  Finally install openHAB2 as either an offline or online distribution.
-  The **offline distribution** comes with all add-ons, the **online distribution** will install additional add-ons from the internet on request.
+  Add the **openHAB 2 Beta Repository** to your systems apt sources list:
 
   ```shell
-  sudo apt-get install openhab2-offline
-  # OR #
-  sudo apt-get install openhab2-online
+  echo 'deb http://dl.bintray.com/openhab/apt-repo2 testing main' | sudo tee /etc/apt/sources.list.d/openhab2.list
   ```
 
 * **Snapshot Release**
 
-  Add the **openHAB 2 Unstable Repository** to your systems apt sources list (will overwrite the existing `openhab2.list`):
+  The snapshot build is created [almost daily](https://bintray.com/openhab/apt-repo2/openhab2#release), and includes the latest changes to the openHAB 2 core and add-ons.
```

Comment:
Give an additional warning? "Only for testing or if instructed by developers" or something like that